Header header

uint8 SETPOINT_TYPE_ATT = 0		# Attitude quaternion setpoint
uint8 SETPOINT_TYPE_RATES = 1	# Angular velocity setpoint
uint8 type

geometry_msgs/Quaternion q
float32 wx
float32 wy
float32 wz




